### PR TITLE
useEffect_ApiData

### PR DESCRIPTION
**Pull request by KannanApp**
0.2 | useEffect and API Data

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState('');
+  
+  const handleDelete = (idx) => {
+    const res = data.filter((item)=> item?.id !== idx);
+    setData(res);
+  }
+
+  useEffect(()=> {
+    fetch("https://jsonplaceholder.typicode.com/posts")
```
Comment (shailendrakanherkar18) [inline src/App.jsx]: you have to write own custom hook useFectch

Comment (KannanApp) [reply src/App.jsx]: Need to create seperate folder for fetching the data?

File: src/App.jsx
Comment (KannanApp) [inline src/App.jsx:1]: MainFile

File: src/App.jsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import './App.css';
+import BlogList from "./Dashboard/BlogList";
+import { TextBox } from "./Components/TextBox";
+import { useFetch } from "./Constants/useFetch";
```
Comment (shailendrakanherkar18) [inline src/App.jsx]: Folder name should be start with small case.

File: src/Constants/useFetch.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import {useState, useEffect} from "react";
+export const useFetch = (url) => {
```
Comment (shailendrakanherkar18) [inline src/Constants/useFetch.js]: empty line expected above useFetech

File: src/Constants/useFetch.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    const [data, setData] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
+    const [error, setError] = useState('');
+    useEffect(() => {
```
Comment (shailendrakanherkar18) [inline src/Constants/useFetch.js:8]: always add empty line to segregate difference between two declaration etc.

File: src/Constants/useFetch.js (hunk 3 above)
Comment (shailendrakanherkar18) [inline src/Constants/useFetch.js]: do you think only url is sufficient for fetch ?

**Pull request merged**